src/Commands/RecreateCommand.php

```diff
@@ -89,7 +89,7 @@ protected function tenantDatabaseExists(Website $website) : bool
                 return true;
             }
         } catch (\Exception $e) {
-            // Surpress exception
+            // Suppress exception
```

Comment:
For any following contributions, make sure not to mix these up with the actual PR. If a PR is not merged this kind of changes might be lost.

Reply:
Should I open a different PR for this?

Reply:
No need 👍 this was just a pointer for next time.